datahub-dao/src/main/java/com/linkedin/datahub/util/CorpUserUtil.java

```diff
@@ -11,6 +11,10 @@
 
 
 public class CorpUserUtil {
+
+    private static final String DEFAULT_AVTAR =
+        "https://raw.githubusercontent.com/linkedin/datahub/master/datahub-web/packages/data-portal/public/assets/images/default_avatar.png";
```

Comment:
Can probably use relative path like /assets/images/default_avatar.png to avoid hard-coded reference to github?

Reply:
Can't directly use "/assets/images/default_avatar.png" since the image file lies under `datahub-web` folder. I can still use relative path.

Reply:
All the static assets under `datahub-web` will be made available to frontend via this route: https://github.com/linkedin/datahub/blob/master/datahub-frontend/conf/routes#L44 so something like `http://localhost:8081/assets/images/default_avatar.png` should work. Using a relative path in this case will avoid hard-coding the hostname part.

Reply:
Thanks for the suggestion.

Reply:
It looks like the picture link set in the method `toUserEntityView` doesn't change the avatar in the `Ownership` tab. I think it's because the `/owners` endpoint returns null `pictureLink`
Here is an e.g.
`{
  "owners": [
    {
      "userName": "fbar",
      "source": "UI",
      "namespace": "urn:li:corpuser",
      "name": "Foo Bar",
      "email": "fbar@linkedin.com",
      "isGroup": false,
      "isActive": true,
      "idType": "USER",
      "type": "DataOwner",
      "subType": null,
      "sortId": null,
      "sourceUrl": null,
      "confirmedBy": "UI",
      "modifiedTime": 0,
      "pictureLink": null
    }
  ],
  "fromUpstream": false,
  "datasetUrn": "urn:li:dataset:(urn:li:dataPlatform:foo,bar,PROD)",
  "lastModified": 0,
  "actor": "fbar"
}`

I will not set it, will let @cptran777 address this issue later. I will limit the scope of this diff to resolving the `NullPointerException`